sml_builder/__init__.py

```diff
@@ -40,3 +44,24 @@ def string_to_paragraph(value):
     """Wraps passed string in <p> tags and converts newlines to <p> pairs"""
     body = escape(value).replace("\n\n", Markup("</p><p>"))
     return Markup(f"<p>{body}</p>")
+
+
+# Method provides a dictionary to the jinja templates, allowing variables
+# inside the dictionary to be directly accessed within the template files
+@app.context_processor
+def set_variables():
+    mkdocs = sml_builder.utils.get_feature_config("docs_integration")
+    navigation = {"navigation": {}}
+    nav_version = (
+        "feature_disabled_navigation"
```

Comment:
I think here is where I'd prefer to just see:
if mkdocs["enabled"] is False
  Original nav code inline
else
  navigation_settings